data/json/items/tools.json

```diff
@@ -2940,38 +2940,218 @@
     "use_action": "MOLOTOV_LIT"
   },
   {
-    "id": "acidbomb",
-    "type": "TOOL",
+    "id": "acidbomb_large",
+    "type": "AMMO",
+    "category": "weapons",
+    "ammo_type" : "thrown",
     "symbol": "*",
-    "color": "yellow",
-    "name": "acid bomb",
-    "description": "This is a fragile container filled with acid.  Throw it to spill out a pool of potent acid.",
-    "price": 5000,
-    "material": "glass",
-    "flags": "ACT_ON_RANGED_HIT",
-    "weight": 650,
+    "color": "i_green",
+    "name": "large acidbomb",
+    "description": "There is enough frothing fluid inside this acidbomb to coat a relatively large area with fields of acid.",
+    "price": 0,
+    "material": ["glass"],
+    "weight": 1640,
     "volume": 2,
-    "bashing": 4,
+    "bashing": 8,
     "to_hit": -1,
+    "flags": ["ACT_ON_RANGED_HIT", "CUSTOM_EXPLOSION"],
```

Comment:
"CUSTOM_EXPLOSION" isn't used nor needed here. You'd need to define an explosion member for this item and then have it explode like a grenade. Explosion actor on the active version and ACT_ON_RANGED_HIT are enough.